src/main/resources/assets/identity/lang/zh_cn.json

```
@@ -1,4 +1,13 @@
 {
   "identity.unlock_entity": "已解锁%s种化身",
   "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃..."
+   "identity.grant_success": "成功将化身%s给予%s。",
```

Invalid syntax here. You forgot to append comma to line 3.
```suggestion
  "identity.revoke_entity": "你的对%s化身的记忆已化为尘埃...",
  "identity.grant_success": "成功将化身%s给予%s。",
```